analysis_runner/cli_analysisrunner.py

```diff
@@ -57,6 +57,14 @@ def add_analysis_runner_args(parser=None) -> argparse.ArgumentParser:
         ),
     )
 
+    parser.add_argument(
+        '--preemptible',
+        required=False,
+        action=argparse.BooleanOptionalAction,
+        default=True,
```

Comment:
How does default true here work if it's not a flag, `--preemptible false`?

Reply:
As far as I understand, it should understand both `--preemptible` and `--no-preemptible` and set the value accordingly.